### DISCUSSION TITLE
Hola !  Busco gente muy animosa para la hackaton :)

### DISCUSSION THREAD

**Discussion by mysticBel** [Launch-X-Latam/MisionFrontEnd#511] [Team Posts]
## Buenas tardes con todos !
Espero podamos aprovechar la experiencia de Launch X y el hackaton que se nos viene y que se encuentren muy bien !!
Soy Maribel , frontend dev y con muchas ganas de seguir aprendiendo y adquiriendo experiencia para ser Fullstack.

Conozco html, css , javascript,  sql server y mysql. Estoy aprendiendo más a fondo React . 
Mi enfoque de desarrollo es con **metodologías agiles**, por lo que estoy muy dispuesta a conformar un equipo responsable y alineado con nuestras historias de usuarios, y objetivos centrados a los clientes (human-centered design).
Estoy a la par aprendiendo ux, por lo que podria tambien aportar desde ese lado.

Tambien me gustaria poder encontrar personas muy entusiasmadas como yo para poder seguir avanzando y aprendiendo en este camino tech, y yo poder ser un apoyo si pudiese..por lo que estoy abierta a formar grupo de estudio y seguir aprendiendo juntos .

Realmente me encantaria que crezcamos en esta experiencia y en lo que sigue !!

Gracias por leerme. 

mi correo de innovaccion es  *lx07729@innovaccion.mx*

**Comment by rauoliv** [comment]
ix10786@innovaccion.mx me uno a tú equipo

**Comment by invitationali** [comment]
Hola Maribel! me interesa formar parte de tu equipo!
Soy diseñador UX/UI y front dev, también me llevo bastante con las metodologías ágiles, así que considero que puedo aportar un poquito por acá!

Mi correo de innovaccion es lx03497@innovaccion.mx

**Comment by IsraelVera** [comment]
Te envié mensaje por teams
Hola buena tarde, me llamó Israel , estoy buscando formar parte de un equipo para el hackaton, la verdad te lo carrera de arquitectura de software pero casi no ejercí o muy poco , y ahorita he estado actualizandome en freecamp, sololearn y pues aquí también espero poder ser parte de tu equipo
Pero igual te lo mando por aquí


Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: Maribel Maza ***@***.***>
Sent: Tuesday, November 8, 2022 2:33:33 PM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-backend] Hola ! Busco gente muy animosa para la hackaton :) (#40)


Buenas tardes con todos !

Espero podamos aprovechar la experiencia de Launch X y el hackaton que se nos viene y que se encuentren muy bien !!
Soy Maribel , frontend dev y con muchas ganas de seguir aprendiendo y adquiriendo experiencia para ser Fullstack.

Conozco html, css , javascript, sql server y mysql. Estoy aprendiendo más a fondo React .
Mi enfoque de desarrollo es con metodologías agiles, por lo que estoy muy dispuesta a conformar un equipo responsable y alineados con nuestras historias de usuarios, y objetivos centrados a los clientes (human-centered design).
Estoy a la par aprendiendo ux, por lo que podria tambien aportar desde ese lado.

Tambien me gustaria poder encontrar personas muy entusiasmadas como yo para poder seguir avanzando y aprendiendo en este camino tech, y yo poder ser un apoyo si pudiese..por lo que estoy abierta a formar grupo de estudio y seguir aprendiendo de uds.

Realmente me encantaria que crezcamos en esta experiencia y en lo que sigue !!

Gracias por leerme.

mi correo de innovaccion es ***@***.******@***.***>

**Comment by NArAr12** [comment]
Hola mi nombre es Natalia, me gustaria ser parte de tu equipo
Mi correo de innovacción lx10738@innovaccion.mx

**Comment by CAMILOITT** [comment]
hola mi nombre es Camilo tambien quiesiera unirme soy desarrollador fullstack, mis conocimentos son: html, css, sass, javascript, react.js, node, express, postgres.
correo de innovacion: lx11002@innovaccion.mx

**Comment by T0ny-dev** [comment]
Hola que tal Maribel me gusta la iniciativa también me apunto, soy también full stack también ando en aprendizaje continuo tengo experiencia igual en javascript , html , css , react , typescript , CI/CD y NodeJs , igual ando enfocado aprendiendo mas react , express y postgrest , breve experiencia en dockers me gustaría contribuir tengo experiencia en Desing Thinking y en hackathones como finalista, también breve experiencia en modelo de negocios. correo innovacion: lx16621@innovaccion.mx

**Comment by CAMILOITT** [comment]
hola mi nombre es Camilo y me interesa formar equipo
correo: lx11002@innovaccion.mx

**Comment by FrankAV27** [comment]
Hola mi nombre es Franklin y quisiera unirme a tu grupo mi correo es: lx09414@innovaccion.mx

**Comment by julianFanjul** [comment]
Hola a todos, me podre sumar a su equipo? soy Julian, de argentina, soy diseñador multimedial con conocimientos en herramientas de diseño y HTML, CSS, REACT, NODE.JS, MONGODB, THREE.JS, entre otros.

Les dejo mi Email si alguien me puede sumar a su equipo estaré agradecido

lx06200@innovaccion.mx

**Comment by CAMILOITT** [comment]
mi nombre es Camilo estoy interesado en hacer el grupo
correo: : lx11002@innovaccion.mx
